package.json

```
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "main": "src/init.lua",
   "types": "src/index.d.ts",
+  "files": ["src/", "LICENSE", "CHANGELOG.md", "README.md"],
```

Can't we just use this?
```suggestion
  "files": ["src"],
```

---

Yep, looks like it! npm automatically includes the following files
```
package.json
README
CHANGES / CHANGELOG / HISTORY
LICENSE / LICENCE
NOTICE
The file in the “main” field
```